src/accessories/alarm.js

```diff
@@ -142,8 +142,19 @@ class SS3Alarm {
     }
 
     async setTargetState(homekitState, callback) {
-        let state = this.TARGET_HOMEKIT_TO_SS3[homekitState];
-        if (this.debug) this.log(`Setting target state to ${state}, ${homekitState}`);
+        var finalState = homekitState;
+
+        // This would be a setting variable
+        //
+        var someDisarmDisableVariable = true
+
+        if (someDisarmDisableVariable && homekitState == "3") {
+          if (this.debug) this.log("Disarming Disabled");
+          var finalState = this.service.getCharacteristic(this.Characteristic.SecuritySystemCurrentState).statusCode;
```

Comment:
This isnt actually doing what I thought it was ... trying to get the currentState